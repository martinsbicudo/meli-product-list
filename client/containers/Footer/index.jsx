import React from 'react';
import Container from '@Components/Container';

const Footer = () => (
  <footer>
    <Container>
      Footer
    </Container>
  </footer>
);

export default Footer;
