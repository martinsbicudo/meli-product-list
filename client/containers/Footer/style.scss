.Footer {
  padding: 2rem 0;
  background-color: map-get($colors, "meli-gallery");

  &Text {
    color: map-get($colors, "meli-dove-gray");
    text-align: center;
    width: 100%;
    font-size: 1.2rem;

    &::after {
      content: "\2764";
      color: map-get($colors, "meli-turbo");
      font-size: 2rem;
    }
  }

  &Link {
    color: map-get($colors, "meli-dodger-blue");
    padding: .5rem 0;
    margin: 0 .5rem;
    text-decoration: none;
    position: relative;

    &::before {
      content: "";
      position: absolute;
      bottom: 0;
      left: 0;
      width: 0;
      height: .2rem;
      background: currentColor;
      transition: .2s width;
    }

    &:hover::before { width: 100%; }
  }
}
