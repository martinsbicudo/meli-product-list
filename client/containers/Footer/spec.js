import Footer from '.';

describe('Containers - <Footer />', () => {
  const createWrapper = () => render(
    <Footer />,
  );

  test('Wrapper Snapshot', () => {
    const { asFragment } = createWrapper();

    expect(asFragment()).toMatchSnapshot();
  });
});
