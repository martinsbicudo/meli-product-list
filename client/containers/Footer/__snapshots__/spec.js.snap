// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Containers - <Footer /> Wrapper Snapshot 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="Container"
    >
      Footer
    </div>
  </footer>
</DocumentFragment>
`;
